.config/forest.dic

```diff
@@ -19,3 +19,12 @@ RPC
 parameterize/D
 timestamp/SM
 synchronizer
+pointer/S
+validator/S
+threadsafe
+blockchain
+CID
+CIDs
```

Comment:
```suggestion
CID/SM
```

Reply:
I tried this at first and it didn't work. Looks like the suffix rules don't work for uppercase words.

Reply:
Weird, they managed to make it work [here](https://gitlab.parity.io/parity/mirrors/polkadot/-/blob/master/scripts/ci/gitlab/lingua.dic) (ie `VRF/SM`).

Reply:
Let keep this in mind and try again in the future. This shouldn't block us.